python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -2767,10 +2763,14 @@ def _store_asset_check_evaluation_planned(
                     run_id=event.run_id,
                     execution_status=AssetCheckExecutionRecordStatus.PLANNED.value,
                     evaluation_event=serialize_value(event),
-                    evaluation_event_timestamp=datetime.utcfromtimestamp(event.timestamp),
+                    evaluation_event_timestamp=self._event_insert_timestamp(event),
                 )
             )
 
+    def _event_insert_timestamp(self, event):
+        # Postgres requires a datetime that is in UTC but has no timezone info
+        return datetime.fromtimestamp(event.timestamp, timezone.utc).replace(tzinfo=None)
```

Comment:
confident this is the same outcome as before? skeptical its under explicit test

Reply:
At first I changed it to just datetime.fromtimestamp(event.timestamp, timezone.utc) and tests failed, which gives me confidence

Reply:
https://buildkite.com/dagster/dagster-dagster/builds/75309#018d8bdc-70df-42d0-b0d5-3183acec0521